spec/lib/fat_zebra/direct_credit_spec.rb

```diff
@@ -39,6 +51,7 @@
 
     it { is_expected.to be_accepted }
     it { expect(direct_credit.reference).to eq(create.reference) }
+    it { expect(direct_credit.amount).to be_instance_of(Float) }
```

Comment:
Would it be a breaking change if we are returning the amount as a `BigDecimal`? I think that should be the preferred type for money amounts.

Reply:
We're relying on the automatic conversion from JSON here, so BigDecimal is not the default type. BigDecimal is really only used if the value is taken to then perform math operations on as it maintains the floating point precision